src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Unloading/UnloadingStepDef.scala

```diff
@@ -739,10 +842,10 @@ class UnloadingStepDef extends BaseStepDef {
   }
 
   And(
-    """^(?:I )?enter number (.+) on 'What is the Customs Union and Statistics CUS code\?' page$"""
-  ) { (answer: String) =>
+    """^(?:I )?enter number (.+) on 'What is the Customs Union and Statistics code for (.+) in house consignment 1\?' page$"""
+  ) { (answer: String, numberOfItems: String) =>
```

Comment:
Should this be "numberOfItems"?

Reply:
I mean should the argument being passed to this method be called numberOfItems or something like CUSCode?